docs/xls-70d-credentials/index.md

```diff
@@ -0,0 +1,47 @@
+# Credentials
+
+The Credentials feature is a set of tools for managing authorization and compliance requirements using the XRP Ledger blockchain, while respecting privacy and decentralization. This feature extends and interconnects with other features of the XRP Ledger including [Deposit Authorization](https://xrpl.org/docs/concepts/accounts/depositauth). The goal of this feature is to streamline the process of compliance checks such as [KYC (Know Your Customer)](https://en.wikipedia.org/wiki/Know_your_customer) requirements and to enable further trust-based applications within the XRP Ledger ecosystem.
+
+The design of Credentials draws from the [W3C Verifiable Credentials standard](https://www.w3.org/TR/vc-data-model-2.0/) and is intended to be compatible to an extent that makes sense in the context of the XRP Ledger. There are some differences in data structures and formatting: for example, the subject of a credential is identified by an XRP Ledger address rather than a URL.
+
+## Overview
+
+_Credentials_ are signed statements that can be stored in the ledger and can attest to a user's identity, legal status, or other status. This feature includes issuance, storage, and verification of credentials directly on the XRP Ledger, while still supporting the privacy needs of users.
+
+This feature set is designed for an ecosystem of parties with the following roles:
+
+- _Authorizers_ who want to limit specific actions or interactions to users who possess the correct credentials.
+- _Issuers_ who provide credentials to users who meet their criteria.
+- _Users_ who obtain credentials from issuers and use those credentials to interact with authorizers.
+
+Each credential applies to a specific user's XRP Ledger account, and is attested by a specific credential issuer. The exact qualifications that a credential entails are open-ended, and defined by their issuers. They can range from trivial to serious, or broad to narrow. Some examples of things a credential could state:
+
+- That a user is not a subject of sanctions by a particular nation or set of nations.
+- That a user has accredited investor status in the US.
+- That a user has reached a specific level in a particular video game.
+
+### Uses
+
+Within the XRP Ledger, you can use Deposit Authorization to automatically allowlist any senders with the correct set of credentials, instead of approving sending accounts individually. Future amendments could further extend credentials to gate access to other features such as holding tokens, participating in lending pools, or contributing to an AMM.
```

Comment:
"Allowlist" is used (intentionally) as a verb here. It's not a code construct, so putting it as `allowlist` is incorrect. But can just use "allow" instead.

re: future amendments, I prefer it as-is because it makes it clear that those aren't features of the protocol as it exists today.